# generic message type for observation specification.
builtin_interfaces/Time stamp
string type
string[] keys
string[] values